code/game/verbs/ooc.dm

```diff
@@ -71,10 +79,13 @@
 						display_name = "[holder.fakekey]/([src.key])"
 					else
 						display_name = holder.fakekey
+
+			var/chat_tag = create_text_tag(chattag, chattagtext, target)
+
 			if(holder && !holder.fakekey && (holder.rights & R_ADMIN) && config.allow_admin_ooccolor && (src.prefs.ooccolor != initial(src.prefs.ooccolor))) // keeping this for the badmins
-				to_chat(target, "<span class='ooc'><span class='[ooc_style]'>" + create_text_tag("ooc", "OOC:", target) + " <font color='[src.prefs.ooccolor]'><EM>[display_name]:</EM> <span class='message'>[msg]</span></font></span></span>")
+				to_chat(target, "<span class='ooc'><span class='[ooc_style]'>" + chat_tag + " <font color='[src.prefs.ooccolor]'><EM>[display_name]:</EM></font></span> <span class='message'>[msg]</span></span>")
```

Comment:
```suggestion
				to_chat(target, "<span class='ooc'><span class='[ooc_style]'>" + chat_tag + " <font color='[src.prefs.ooccolor]'><EM>[display_name]:</EM></span> <span class='message'>[msg]</span></font></span>")
```

Reply:
```
<span class='ooc'>
  <span class='[ooc_style]'>tag
    <font color='[src.prefs.ooccolor]'>
      <EM>[display_name]:</EM>
    </font>
  </span>
  <span class='message'>[msg]</span>
</span>
```

```
<span class='ooc'>
  <span class='[ooc_style]'>tag
    <font color='[src.prefs.ooccolor]'>
      <EM>[display_name]:</EM>
    </span>
    <span class='message'>[msg]</span>
  </font>
</span>
```

This change makes the HTML improper. Tags must be closed in the order they're opened in.

Reply:
Also the idea for this PR was originally to remove the colour from the full line and reduce it just to the name. The chat tags kinda took over but the intention is to make everything after `Ckey:` be default colored, like on other servers.

Reply:
The chat tags were a way to make admins more visible at a glance compared to regular players.